smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/json/MemberShapeEncodeGenerator.kt

```diff
@@ -1,270 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-package software.amazon.smithy.swift.codegen.integration.serde.json
-
-import software.amazon.smithy.model.knowledge.NullableIndex
-import software.amazon.smithy.model.shapes.BlobShape
-import software.amazon.smithy.model.shapes.CollectionShape
-import software.amazon.smithy.model.shapes.MapShape
-import software.amazon.smithy.model.shapes.MemberShape
-import software.amazon.smithy.model.shapes.Shape
-import software.amazon.smithy.model.shapes.ShapeType
-import software.amazon.smithy.model.shapes.StringShape
-import software.amazon.smithy.model.shapes.TimestampShape
-import software.amazon.smithy.model.traits.BoxTrait
-import software.amazon.smithy.model.traits.EnumTrait
-import software.amazon.smithy.model.traits.SparseTrait
-import software.amazon.smithy.model.traits.StreamingTrait
-import software.amazon.smithy.model.traits.TimestampFormatTrait
-import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
-import software.amazon.smithy.swift.codegen.SwiftWriter
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.MemberShapeEncodeGeneratable
-import software.amazon.smithy.swift.codegen.integration.serde.TimestampEncodeGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.TimestampHelpers
-import software.amazon.smithy.swift.codegen.integration.serde.getDefaultValueOfShapeType
-import software.amazon.smithy.swift.codegen.model.hasTrait
-import software.amazon.smithy.swift.codegen.model.isBoxed
-import software.amazon.smithy.swift.codegen.removeSurroundingBackticks
-import software.amazon.smithy.swift.codegen.utils.toLowerCamelCase
-
-/*
-Includes functions to help render conformance to Encodable protocol for shapes
- */
-abstract class MemberShapeEncodeGenerator(
```

Comment:
JSON member shape encode is deleted and the unified implementation is used instead.